src/main/java/com/rylinaux/plugman/util/PluginUtil.java

```diff
@@ -345,6 +352,45 @@ public static boolean isIgnored(String plugin) {
         return false;
     }
 
+    public static PluginDescriptionFile getPluginDescription(File file) throws InvalidDescriptionException {
```

Comment:
I had to add this method as paper hides it. This is a direct copy from the original workings.

Reply:
Once the paper PR is merged, you will no longer need to duplicate this logic.